### PR TITLE
test for landing page

### PR DESCRIPTION
**Pull request by liruochen1998**

### CODE REVIEW

File: src/components/LandingPage/landingPage.test.js
Comment (carolinepar) [inline src/components/LandingPage/landingPage.test.js:12]: Thanks for taking care of this test! Should we test that the button causes the main page to render?

**Review by carolinepar — CHANGES_REQUESTED** [review]

File: src/components/LandingPage/landingPage.test.js
Comment (carolinepar) [reply src/components/LandingPage/landingPage.test.js:12]: Or is this something that is good enough to test manually? Now that I think about it I'm sure that's the case, but want to double check in case.

Comment (liruochen1998) [reply src/components/LandingPage/landingPage.test.js:12]: let me see whether there's an easy why. If not, I don't think we should spend extra time on it

Comment (carolinepar) [reply src/components/LandingPage/landingPage.test.js:12]: I agree. After I looked at this one I wondered the same thing about the help button and googled testing hover buttons and the response was basically 'why bother when you can test it manually easily' and I think that applies to this situation too

Comment (Jaega) [reply src/components/LandingPage/landingPage.test.js:12]: So what's our plan, should we remove this test function?

Comment (liruochen1998) [reply src/components/LandingPage/landingPage.test.js:12]: I just reviewed the code, there's no easy way we can test it since the onclick is from the parent actually and everything in this component is in render(). So I think we can merge this. Answer @Jaega 's question. I think we are trying to aim for a minimum test (whether it can render) for every component, so here maybe we just test whether it can render and that's all. I believe since everything's in render(), it might show a 100% coverage for this file actually.

**Comment by liruochen1998** [comment]
we can go ahead and merge this after y'all 's approvals. @Jaega @carolinepar

File: src/components/LandingPage/landingPage.test.js
Comment (carolinepar) [reply src/components/LandingPage/landingPage.test.js:12]: I think there's a way to do it with mocks so I can give it a try later on, but for now we can go ahead and run with this test.

**Review by carolinepar — APPROVED** [review]

File: src/components/LandingPage/landingPage.test.js
Comment (liruochen1998) [reply src/components/LandingPage/landingPage.test.js:12]: feel free to make any changes if you think you can find a way to make that work. Thanks!

**Review by Jaega — APPROVED** [review]

**Pull request merged**
